src/functionsShellCommandAction.ts

```diff
@@ -99,7 +98,7 @@ export const actionFunction = async (options: Options) => {
           if (emulator.emulatedFunctions.includes(trigger.id)) {
             const localFunction = new LocalFunction(trigger, emulator.urls, emulator);
             const triggerNameDotNotation = trigger.name.replace(/-/g, ".");
-            _.set(context, triggerNameDotNotation, localFunction.call);
+            context[triggerNameDotNotation] = localFunction.makeFn();
```

Comment:
Can you double check that this still works as expected for functions with -'s in the name? `_.set` will treat `triggerNameDotNotation` as a path (https://www.geeksforgeeks.org/lodash-_-set-method/)  but `context[triggerNameDotNotation` will set it as a single top level attribute.

Reply:
great catch!!

Reply:
Reverting back to using lodash :(